testing/ostest/prioinherit.c

```diff
@@ -31,7 +31,7 @@
 #include <stdlib.h>
 #include <unistd.h>
 
-#ifdef CONFIG_ARCH_SIM
+#ifdef CONFIG_SIM_WALLTIME_SLEEP
```

Comment:
merge to previous patch

Reply:
I think that change is consistent as it changes `#ifdef CONFIG_ARCH_SIM` to `#ifdef CONFIG_SIM_WALLTIME_SLEEP` in all related places